apps/tlon-mobile/src/lib/api.ts

```diff
@@ -34,10 +34,18 @@ export function configureClient({
   shipName,
   shipUrl,
   onReset,
```

Comment:
Should we remove `onReset`? Not clear on what it's actually doing for us after the convo yesterday.

Reply:
We can if you'd prefer, I don't feel strongly. It's not there to handle anything explicit right now, at the moment it's only relevant if we ourselves manually attempt to reset it. 

Would rather leave it in defensively though since if the client ever does emit that, we can be certain we need to resub.

Reply:
My take would be remove the param, but pipe all the "potential reset" events through one handler. Up to you tho, not blocking! If that doesn't make sense, merge away.